layouts/shortcodes/ee-download.html

```diff
@@ -18,7 +18,7 @@
 
   camDownloads.servers = {};
   {{ range $info := sort $.Page.Params.downloads.servers "weight" }}
-  camDownloads.servers['{{ $info.path }}'] = '{{ $info.name }}';{{ end }}
+  camDownloads.servers['{{ cond (not (not $info.id)) $info.id $info.path }}'] = {name:'{{ $info.name }}',path:'{{ $info.path }}'};{{ end }}
```

Comment:
🙃 This is accounting for servers not having an  ID, right? I think it's simpler to assume all servers have one by convention. The required changes in static docs are simple & small and it improves readability/maintainability here.

Reply:
It does, it uses `path` if there is no `id`. But it makes sense, we don't have that many servers defined. Let's add an id everywhere!